diligent-academy-assessments/backend/database-challenge/README.md

```diff
@@ -0,0 +1,22 @@
+# Database challenge
+
+## Principles and rules
+
+- Please use the prepared `.sql` files to solve the tasks.
+
+## Tasks
+
+1. Create the student table with the following fields: name and email. What fields would you create, what would be the
+   type of them?
+
+
+2. Create the class table, each class has a name, and it have to be **unique**.
+   Every student can be in only one class.
+   How would you modify the table if we had a lot of read operations by searching on the class name.
+
+3. How would you query every student in a given class by name?
+
+4. Create the subject table, each subject has a name. Every student can attend multiple subjects.
+   Where would you store the grades of the students?
```

Comment:
Should the subject be part of a class or totally freestanding.

Reply:
Maybe we are thinking about different things when we talk about classes. I'm thinking about the specific class you are in in a given grade, like how I was once in 9th grade and in class C. I could not find a better word the class.
Subjects are like math and english and stuff.

Reply:
I see, that's okay.